src/lib/mina_caqti/mina_caqti.ml

```diff
@@ -2,18 +2,68 @@
 
 open Async
 open Core_kernel
-open Caqti_async
 open Mina_base
 
 (* custom Caqti types for generating type annotations on queries *)
-type _ Caqti_type.field +=
-  | Array_nullable_int : int option array Caqti_type.field
+let find_req t u s = Caqti_request.Infix.(t ->! u) s
```

Comment:
nit: use of these utilities is inconsistent. See changes in archive/lib/processor.ml